vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1316,6 +1316,16 @@ EXPORT(int, sceGxmAddRazorGpuCaptureBuffer) {
     return UNIMPLEMENTED();
 }
 
+void update_viewport(renderer::State &state, SceGxmContext *context) {
```

Comment:
no better using static void ? 🤔

Reply:
You're right. There's no reason for that function not to be static.